fluss-client/src/main/java/com/alibaba/fluss/client/write/RecordAccumulator.java

```diff
@@ -232,6 +243,10 @@ public RecordAppendResult append(
      * </pre>
      */
     public ReadyCheckResult ready(Cluster cluster) {
+        return ready(cluster, batchTimeoutMs);
+    }
+
+    public ReadyCheckResult ready(Cluster cluster, long timeoutMs) {
```

Comment:
why do we need to provide this method? only testing purpose? If yes, why not use the `batchTimeoutMs` parameter in tests?